.github/workflows/python-app.yml

```diff
@@ -0,0 +1,44 @@
+# Based on the `python-app.yml` template from GitHub actions:
+# https://github.com/actions/starter-workflows/blob/main/ci/python-app.yml
+#
+# Adapted by Diego Ramirez.
+
+name: Python application tests
```

Comment:
I added this accidentaly on this branch :disappointed:. Anyway, this can help us to test before merging.

Reply:
Seems like it's working. `flake8` failed with some messages. I can fix that on another pull request.